tracer/src/Datadog.Trace/Agent/MessagePack/SpanModel.cs

```diff
@@ -31,17 +31,21 @@ internal readonly struct SpanModel
 
     public readonly bool IsFirstSpanInChunk;
 
+    public readonly bool IsInFirstChunkOfBuffer;
```

Comment:
If this is the same as `TraceChunk.IsFirstChunkInBuffer`, then we don't need to add it. The span has a reference to the trace chunk so you can access `span.TraceChunk.IsFirstChunkInBuffer`.